### DISCUSSION TITLE
Does it support nextjs static deployment?

### DISCUSSION THREAD

**Discussion by zjy365** [fuma-nama/fumadocs#1008] [Q&A] [answered]
Does it support nextjs static deployment? The configuration is as follows
const config = {
output: 'export',
reactStrictMode: true,
}

**Comment by zjy365** [comment]
I personally feel that it is not supported, because I have seen the use of API Routes and Middleware

**Comment by fuma-nama** [comment, marked as answer]
Yes, our main site is also using static export. Search can be integrated with 3rd party services or using the static Orama search (see docs). You still need a middleware for i18n, but it works for docs sites without i18n support

**Reply by zjy365** [reply]
Thanks for your reply, I will check out the documentation
